docs/android/manifest.md

```diff
@@ -0,0 +1,246 @@
+
+## Intent actions and features
+
+The Android manifest file provides a way for you to describe the capabilities of your app. This is achieved via [Intents](https://developer.android.com/guide/topics/manifest/intent-filter-element.html) and the <xref:Android.App.IntentFilterAttribute>. You can specify which actions are appropriate for your activity with the <xref:Android.App.IntentFilterAttribute> constructor, and which categories are appropriate with the <xref:Android.App.IntentFilterAttribute.Categories> property. At least one activity must be provided, which is why activities are provided in the constructor. An `[IntentFilter]` can be provided multiple times, and each use results in a separate `<intent-filter/>` element within the `<activity/>`:
+
+```csharp
+using Android.App;
+using Android.Content;
+
+namespace MyMauiApp;
+
+[Activity(Label = "My Maui App", MainLauncher = true)]
+[IntentFilter(new[] {Intent.ActionView},
+    Categories = new[] {Intent.CategorySampleCode, "my.custom.category"})]
+public class MyActivity : Activity
+{
+}
+```
+
+This example produces the following XML fragment:
+
+```xml
+<activity android:label="My Maui App"
+          android:name="crc64bdb9c38958c20c7c.MainActivity">
+  <intent-filter>
+    <action android:name="android.intent.action.MAIN" />
+    <category android:name="android.intent.category.LAUNCHER" />
+  </intent-filter>
+  <intent-filter>
+    <action android:name="android.intent.action.VIEW" />
+    <category android:name="android.intent.category.SAMPLE_CODE" />
+    <category android:name="my.custom.category" />
+  </intent-filter>
+</activity>
+```
+
+### Application element
+
+The Android manifest file also provides a way for you to declare properties for your entire app. This is achieved via the `<application>` element and its counterpart, the <xref:Android.App.ApplicationAttribute>. Typically, you declare `<application>` properties for your entire app and then override these properties as required on an activity basis.
+
+For example, the following `Application` attribute could be added to *MainApplication.cs* to indicate that the app can be debugged, that its user-readable name is "My Maui App", and that it uses the `Maui.SplashTheme` style as the default theme for all activities:
+
+```csharp
+using Android.App;
+using Android.Runtime;
+
+namespace MyMauiApp;
+
+[Application(Debuggable = true, Label = "My Maui App", Theme = "@style/Maui.SplashTheme")]
+public class MainApplication : MauiApplication
+{
+      public MainApplication(IntPtr handle, JniHandleOwnership ownership)
+             : base(handle, ownership)
+      {
+      }
+
+      protected override MauiApp CreateMauiApp() => MauiProgram.CreateMauiApp();
+}
+
+```
+
+This declaration causes the following XML fragment to be generated in *obj\\Debug\\net7.0-android\\AndroidManifest.xml*:
+
+```xml
+<application android:label="MyMauiApp" android:debuggable="true" android:theme="@style/Maui.SplashTheme" ...>
+```
```

Comment:
We could still give this example, but maybe we just say Debug apps do this and Release apps don't.